spec/dash-modal/view.coffee

```diff
@@ -86,18 +83,26 @@ describe 'DashModal.View', ->
     _m.show()
     expect(_m.isVisible()).toBeTruthy()
 
-  it 'hides when .modal is clicked when shouldAllowClose is true', ->
-    template = '<div>Hello</div>'
-    _m = modal(view(template), 'test', true).show()
+  it 'hides when the overlay (.modal) is clicked when shouldCloseOnOverlay is true', ->
+    _m = new DashModal.View
```

Comment:
I would try to continue using a helper method. There are two reason:

1. Easier to update tests if/when there are changes to `DashModal.View`.
2. It can be useful to default values that are not passed in, so the tests only specify the values they care about.

Maybe refactor to be:

```coffee
_m = modalView
  shouldCloseOnOverlay: true
  router: router
```